### PR TITLE
doc(atomic): edits to descriptions

### PR DESCRIPTION
**Pull request by dlutzCoveo**
Hello. This PR has many small changes based on feedback from a PR on the generated docs. Much of it is adding periods at the end of descriptions, and correcting phrasing, etc. Additionally, the generated Stencil file, ands README files are included.  There are a few items with comments that require clarity.

### CODE REVIEW

File: packages/atomic/src/components/atomic-query-summary/atomic-query-summary.tsx [hunk 1]
```diff
@@ -22,14 +22,14 @@ interface Summary {
...
- * @part no-results - The container when there are no results
- * @part duration - The duration container
- * @part highlight - The summary highlights
- * @part placeholder - The initialization placeholder
+ * @part container - The container for the whole summary.
+ * @part results - The container for the results.
+ * @part no-results - The container for when there are no results.
+ * @part duration - The container for the duration.
+ * @part highlight - The summary highlights.
+ * @part placeholder - The query summary placeholder used while the search interface is initializing.
```
Comment (dlutzCoveo) [inline packages/atomic/src/components/atomic-query-summary/atomic-query-summary.tsx:32]: Is this an accurate description of placeholder?

File: packages/atomic/src/components/facets/atomic-category-facet/readme.md [hunk 2]
```diff
@@ -10,41 +10,41 @@ A hierarchical category facet component. It is displayed as a facet in desktop b
...
 | `numberOfValues`      | `number-of-values`     | The number of values to request for this facet. Also determines the number of additional values to request each time this facet is expanded, and the number of values to display when this facet is collapsed. | `number`                          | `8`             |
-| `sortCriteria`        | `sort-criteria`        | The sort criterion to apply to the returned facet values. Possible values are 'alphanumeric', and 'occurrences''.                                                                                              | `"alphanumeric" \| "occurrences"` | `'occurrences'` |
+| `sortCriteria`        | `sort-criteria`        | The sort criterion to apply to the returned facet values. Possible values are `alphanumeric`, and `occurrences`.                                                                                               | `"alphanumeric" \| "occurrences"` | `'occurrences'` |
```
Comment (dlutzCoveo) [inline packages/atomic/src/components/facets/atomic-category-facet/readme.md:20]: I believe the types here are incorrect, but wasn't sure how far to drill down. Should the type of `sort-criteria` in this table reduce to primitive types?

**Comment by dlutzCoveo** [comment]
The generated docs from Stencil's output with edits in this PR are staged here:
https://docsstaging.coveo.com/DOC-9119-generate-atomic-v0.13.1/atomic

File: packages/atomic/src/components/facets/atomic-category-facet/readme.md (hunk 2 above)
Comment (samisayegh) [reply packages/atomic/src/components/facets/atomic-category-facet/readme.md:20]: Yep, a developer would have to specify one of these values, so seeing them in the documentation is helpful.

File: packages/atomic/src/components/atomic-query-summary/atomic-query-summary.tsx (hunk 1 above)
Comment (samisayegh) [reply packages/atomic/src/components/atomic-query-summary/atomic-query-summary.tsx:32]: Yep, maybe I would change the word `used` -> `shown`?

```
The query summary placeholder shown while ...
```

**Review by samisayegh — APPROVED** [review]

**Review by ThibodeauJF — APPROVED** [review]

File: packages/atomic/src/components/facets/atomic-category-facet/readme.md (hunk 2 above)
Comment (dlutzCoveo) [reply packages/atomic/src/components/facets/atomic-category-facet/readme.md:20]: OK. I'm going to adjust this on the docs side.

**Pull request merged**